src/ray/raylet/node_manager.cc

```diff
@@ -1552,23 +1635,32 @@ void NodeManager::HandleTaskReconstruction(const TaskID &task_id) {
 }
 
 void NodeManager::ResubmitTask(const Task &task) {
-  if (task.GetTaskSpecification().IsActorTask()) {
-    // Actor reconstruction is turned off by default right now.
-    const ActorID actor_id = task.GetTaskSpecification().ActorId();
-    auto it = actor_registry_.find(actor_id);
-    RAY_CHECK(it != actor_registry_.end());
-    if (it->second.IsAlive()) {
-      // If the actor is still alive, then do not resubmit.
-      RAY_LOG(ERROR) << "The output of an actor task is required, but the actor may "
-                        "still be alive. If the output has been evicted, the job may "
-                        "hang.";
-      return;
+  RAY_LOG(DEBUG) << "Resubmitting task " << task.GetTaskSpecification().TaskId();
```

Comment:
We should keep the message on `INFO` and make sure to only print it if the task really does get resubmitted.

Reply:
Thanks, I patched the code from the old PR and didn't notice this log level was changed.